adminpages/wizard/done.php

```diff
@@ -0,0 +1,97 @@
+			}
+			esc_html_e( 'Here are some recommended Add Ons for your business.', 'paid-memberships-pro' );
+			?>
+		</p>
+		<div style="text-align:center;">
+		<?php
+			// Get some Add On recommendations and only show 3.
+			$random_addon = array_rand( $addon_list, 3 );
+			foreach( $random_addon as $key ) {
+				$addon_slug = $addon_list[$key];
+				$addon = pmpro_getAddonBySlug( $addon_slug );
+
+				$title = str_replace( 'Paid Memberships Pro - ', '', $addon['Title'] );
+				$link = $addon['PluginURI'];
+				$icon = pmpro_get_addon_icon( $addon_slug );
+				?>
+				<div class="pmpro-wizard__col3">
+				<p><a href="<?php echo esc_url( $link ); ?>" target='_blank' rel='nofollow'><img src="<?php echo esc_url( $icon ); ?>" width="70%"/><br>
+				<strong><?php esc_html_e( $title ); ?></strong></a>
+				</p>
+			</div>
+				<?php
+			}
+		?>
+		</div> <!-- end of center alignment -->	
+		<p class="pmpro-wizard__textbreak"><?php esc_html_e( 'OR', 'paid-memberships-pro' ); ?></p>
+		<div class="pmpro-wizard__col">
+			<p><span class="pmpro-wizard__subtitle"><?php esc_html_e( 'More functionality', 'paid-memberships-pro' ); ?></span><br>
+			<?php esc_html_e( 'Add more features to your membership site.', 'paid-memberships-pro' ); ?></p>
+		</div>
+		<div class="pmpro-wizard__col">
+			<a href="<?php echo esc_url( admin_url( 'admin.php?page=pmpro-addons' ) ); ?>" class="button button-primary button-hero"><?php esc_html_e( 'View available Add Ons', 'paid-memberships-pro' ); ?></a>
+		</div>
+
+		<?php if ( $configure_payment ) { ?>
+			<div class="pmpro-wizard__col">
+				<p><span class="pmpro-wizard__subtitle"><?php esc_html_e( 'Payments', 'paid-memberships-pro' ); ?></span><br>
+				<?php esc_html_e( 'Finish configuring your payment gateway.', 'paid-memberships-pro' ); ?></p>
+			</div>
+			<div class="pmpro-wizard__col">
+				<a href="<?php echo esc_url( admin_url( 'admin.php?page=pmpro-paymentsettings' ) ); ?>" class="button button-hero"><?php esc_html_e( 'View settings', 'paid-memberships-pro' ); ?></a>
+			</div>
+		<?php } ?>
+
+		<div class="pmpro-wizard__col">
+			<p><span class="pmpro-wizard__subtitle"><?php esc_html_e( 'Documentation', 'paid-memberships-pro' ); ?></span><br>
+			<?php esc_html_e( 'Not sure where to start, take a look at our documentation.', 'paid-memberships-pro' ); ?></p>
+		</div>
+		<div class="pmpro-wizard__col">
+			<a href="https://www.paidmembershipspro.com/documentation/?utm_source=plugin&utm_medium=setup-wizard&utm_campaign=wizard-done&utm_content=view-docs" target="_blank" class="button button-hero"><?php esc_html_e( 'View docs', 'paid-memberships-pro' ); ?></a>
+		</div>
+	</div>
+	<script>
+		jQuery(document).ready(function(){
+			const run_confetti = () => {
+			setTimeout(function() {
+				confetti.start()
+			}, 1000); //start after 1 second.
+			setTimeout(function() {
+				confetti.stop()
+			}, 4000); //Stop after 4 seconds.
+		};
+			run_confetti();
+		});
+	</script>
+</div> <!-- end pmpro-wizard__step-5 -->
```

Comment:
We could clean up some options here that won't be needed again:

pmpro_wizard_collect_payments, pmpro_wizard_skip_payment